ubg/src/pages/Game.js

```diff
@@ -233,13 +233,16 @@ function FavoriteButton({usersCollection, game}) {
   const [favorite, setFavorite] = useState(inFavorites(userGames, game));
 
   return (
-    <Button
-      variant='contained'
-      color='primary'
-      onClick={() => addFavorite(
-          userGames, usersCollection, game, favorite, setFavorite, user.uid)}>
-      {favorite ? 'Delete from favorites' : 'Add to favorites'}
-    </Button>
+    {user} ?
+    (
+      <Button
+        variant='contained'
+        color='primary'
+        onClick={() => addFavorite(
+            userGames, usersCollection, game, favorite, setFavorite, user.uid)}>
+        {favorite ? 'Delete from favorites' : 'Add to favorites'}
+      </Button>
+    ) : ' '
```

Comment:
I think instead of '  ' you can use null, because I think that way it actually renders '  '

Reply:
alright I'll try that out